### DISCUSSION TITLE
Release 1.6.0

### DISCUSSION THREAD

**Discussion by BitMaker-hub** [BitMaker-hub/NerdMiner_v2#109] [Announcements]
**¡**We are ready to join low diff pools**!**

Miner **mandatori update**. 
All miners should uptade to this version to be compatible with low diff pools:

- Solved bugs found on merkle root calculations. (@golden-guy)
- Solved issues about difficulty calculation (@golden-guy)
- Tested on low difficulty pools. (NerdMinerTeam)
- Changed the default pool to http://public-pool.airdns.org/
- Added new notes to readme about supported pools (@BitMaker-hub)
- Added new custom sha tests to use on next versions (@BitMaker-hub)

You can also build it easily using the flasher tool:
https://bitmaker-hub.github.io/diyflasher/

<hr /><em>This discussion was created from the release <a href='https://github.com/BitMaker-hub/NerdMiner_v2/releases/tag/Release1.6.0'>Release 1.6.0</a>.</em>

**Comment by therealcharon** [comment]
Updated today on 1.6.0 - the miner crashed every 5 to 10 minutes. Previous version don't.

**Reply by L0wB0b** [reply]
My four miners running now since 2 hours without problems. I did a clean install (erase flash, install image and configure new). Maybe this will help you.

**Reply by ROSITO** [reply]
I had update 4 miners, 2 run good, 2 restart every 10/20 minutes

**Reply by 570RM7R4C3R** [reply]
Me too, i have 3 nerd miners v2, updated, and everything is ok, about 2 days uptime no issues.
